libs/driver/magento/src/graphql/cacheable-operations/apollo-cacheable-operations-link-generator.ts

```diff
@@ -0,0 +1,26 @@
+import { Inject, Injectable } from '@angular/core';
+import { ApolloLink } from '@apollo/client/core';
+import { DaffApolloCacheableOperationLinkGenerator } from '@daffodil/core/graphql';
+
+import { DAFF_MAGENTO_CACHEABLE_OPERATIONS } from './cacheable-operations-token';
+
+/**
+ * A service that will convert cacheable apollo operations into a format that Magento will understand as cacheable.
+ */
+@Injectable({ 
+	providedIn: 'root'
+})
+export class DaffMagentoApolloCacheableOperationsLinkGenerator implements DaffApolloCacheableOperationLinkGenerator {
+	constructor(
+		@Inject(DAFF_MAGENTO_CACHEABLE_OPERATIONS) private apolloGetRequests: string[],
+	) {}
+
+	getLink(): ApolloLink {
+		return new ApolloLink((operation, forward) => {
+			if(this.apolloGetRequests.includes(operation.operationName)) {
```

Comment:
`includes` isn't supported by IE, can you check whether or not typescript will compile this appropriately to `indexOf`?

Reply:
I don't think it does. changed to `indexOf` usage